### ISSUE TITLE
Integrate with form-feed-mode

### ISSUE THREAD

**Issue by mina86** [browse-kill-ring/browse-kill-ring#59]
Emacs has `form-feed-mode` which displays form feed characters on lines by themselves to be displayed as horizontal lines. Based on that, I’ve tried doing:

```
(setq browse-kill-ring-separator "\x0C")  ; form feed
(add-hook** 'browse-kill-ring-hook 'form-feed-mode))
```

and while form feed character is inserted as separator and `form-feed-mode` is enabled in the `*Kill Ring*` buffer, the form feed characters aren’t displayed as lines.

I suspect this is because `browse-kill-ring` sets face on the separator which interferes with `form-feed-mode`.

**Comment by rnkn** [comment]
Try https://github.com/purcell/page-break-lines

**Comment by T-J-Teru** [comment]
This should be fixed in commit b36d5a9.
